backend/section/views.py

```diff
@@ -51,11 +55,18 @@ def get_section(request: HttpRequest, section_id: int) -> Section:
         response = FileResponse(open(filepath, "rb"))
 
         # Header is required to make seeking work in Chrome
-        response['Accept-Ranges'] = 'bytes'
+        response["Accept-Ranges"] = "bytes"
 
         # Response may log a ConnectionResetError on the development server
         # This has no effect on serving the file
         return response
 
     except Section.DoesNotExist:
         raise Http404("Section does not exist")
+
+
+@api_view(["GET"])
+def playlists(request):
+    """Return a list of all playlists"""
+    playlists = [{"id": playlist.id, "name": playlist.name} for playlist in Playlist.objects.all()]
+    return Response(playlists)
```

Comment:
This is used by the experiment form in react to be able to show all playlists and pick one from a select input